cmd/svcat/command/namespaced.go

```diff
@@ -16,23 +16,70 @@ limitations under the License.
 
 package command
 
-// Namespaced is the base command of all svcat commands that are namespace scoped.
-type Namespaced struct {
+import "github.com/spf13/pflag"
+
+// HasNamespaceFlags represents a command that can be scoped to a namespace.
+type HasNamespaceFlags interface {
+	Command
+
+	// ApplyNamespaceFlags persists the namespace-related flags:
+	// * --namespace
+	// * --all-namespaces
+	ApplyNamespaceFlags(flags *pflag.FlagSet)
+}
+
+// NamespacedCommand is the base command of all svcat commands that are namespace scoped.
+type NamespacedCommand struct {
```

Comment:
Why rename Namespaced => NamespacedCommand?

Reply:
When we were refining how we wanted the interfaces to look working on Waitable, we went with putting the more generic name "Waitable" or "Namespaced" on the interface, and using "*Command" for the implementation struct.

Reply:
lol oh wait... Now I'm confused... 🤔 

The original reason that I said isn't true anymore, the interface is called HasNamespaceFlags...

I don't terribly care about the names, just want to be consistent, and this is what waitable did. If you have strong feels, let me know.

Reply:
I strongly feel I shouldn't be reviewing this PR, lol.

The opinion is I find the usages which end up as "command.ThingCommand" overly long.

Reply:
> I strongly feel I shouldn't be reviewing this PR, lol.

I've noticed that people are being added automatically to PRs when I open them lately. Is that a prow thing?

Reply:
> The opinion is I find the usages which end up as "command.ThingCommand" overly long.

Agreed, you cool with me fixing up both in this PR? to be "Waitable"+"HasWaitFlags" and "Namespaced"+"HasWaitFlags"?